recipes/windmapper/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "windmapper" %}
+{% set version = "2.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/windmapper-{{ version }}.tar.gz
+  sha256: 45df92b712677d083547c51b4b9a15cf4f55dc996991af3512b7d0844c70fed6
+
+build:
+  entry_points:
+    - windmapper = windmapper:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  host:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - make
+    - python
+    - scikit-build-core
+    - pip
+    - cmake >=3.16
+  run:
+    - python
+    - numpy
+    - scipy
+    - elevation
+    - pyproj
+    - tqdm
+    - rasterio
+    - cloudpickle
+    - gdal
+    - mpi4py
+    - windninja
+
+test:
+  imports:
+    - windmapper
+  requires:
+    - pip
+
```

Comment:
```suggestion
  requires:
    - pip
    - python {{ python_min }}
  commands:
    - pip check
```

Reply:
There is currently no `--help`

Reply:
Is there any commad that we can use to test the entry-point?

Reply:
It doesn't at the moment. Is that required?

Reply:
Not required, but nice to have.

Reply:
I'll make sure to add it in the future